tools/k8s/devcluster.yaml

```diff
@@ -10,7 +10,7 @@ stages:
       password: postgres
       container_name: determined_db
       image_name: "postgres:10.14"
-      data_dir: det-postgres
+      data_dir: ~/.postgres-minikube
```

Comment:
ditto; are you changing the data directory in this PR? or is this a misplaced change?

Reply:
Actually meant to remove this from the PR, Sorry about that!